gpt_index/embeddings/base.py

```diff
@@ -76,6 +82,16 @@ def _get_text_embeddings(self, texts: List[str]) -> List[List[float]]:
         result = [self._get_text_embedding(text) for text in texts]
         return result
 
+    async def _aget_text_embeddings(self, texts: List[str]) -> List[List[float]]:
+        """Asynchronously get text embeddings.
+
+        By default, this is a wrapper around _aget_text_embedding.
+        Can be overriden for batch queries.
+
+        """
+        result = [await self._aget_text_embedding(text) for text in texts]
```

Comment:
This looks sequential. Shouldn't this be more like
```suggestion
        #               ▼ Depends on `asyncio` runtime.
        result = await asyncio.gather(*[
            self._aget_text_embedding(text) 
            for text in texts
        ])
```
?

Reply:
Yeah you're right, though this gets overridden anyways, but agreed I'll make the change